<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_chessboard1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_chessboard2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_chessboard3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_chessboard4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_chessboard5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_chessboard6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_chessboard7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>400</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_chessboard8">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>400</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_chessboard9">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>400</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>140</y>
     <width>160</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_compFirst">
      <property name="text">
       <string>电脑先手</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_humanFirst">
      <property name="text">
       <string>人先手</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
